### PR TITLE
Issue #1706 Refactor `profile delete` messages

### LINKED ISSUE #1706: Confusing message for 'minishift profile delete'
When deleting a profile with no existing VM, the output is confusing:

    $ minishift profile delete ttt --force
    VM for profile 'ttt' does not exist
    Deleted:  /Users/hardy/.minishift/profiles/ttt
    Profile 'ttt' deleted successfully

I am confused about the "VM for profile 'ttt' does not exist". What is this supposed to mean? I think this can be removed. 

I also suggest to combine the two remaining lines of output into one:

"Profile 'ttt' (/Users/hardy/.minishift/profiles/ttt) deleted successfully."

Please, also end messages with a full stop.

### PR DESCRIPTION
**Pull request by ulyssessouza**
Fixes #1706 

- Consolidate delete messages

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@ulyssessouza We are currently trying to come to conclusion around what we should show in the `profile delete` output. Once we done with the discussion we can merge this PR with relevant changes.

**Comment by LalatenduMohanty** [comment]
@ulyssessouza As per the discussion in #1706 this should be the output of the `profile delete` command
```
$ minishift profile delete ttt
Will remove the VM and all the related artifacts. Do you want to continue [y/N]?: y
Profile 'ttt' deleted successfully.
```
```
$ minishift profile delete ttt --force
Profile 'ttt' deleted successfully.
```
If the VM exists
```
$ minishift profile delete foo --v 2 
Will remove the VM and all the related artifacts. Do you want to continue [y/N]?: y
Deleted: Minishift VM 'foo'
Deleted:  /Users/lmohanty/.minishift/profiles/foo
Profile 'foo' deleted successfully
```
If the VM does not exist
```
$ minishift profile delete foo --v 2 
Will remove the VM and all the related artifacts. Do you want to continue [y/N]?: y
Deleted:  /Users/lmohanty/.minishift/profiles/foo
Profile 'foo' deleted successfully
```

File: pkg/minikube/cluster/cluster.go [hunk 1]
```diff
@@ -129,6 +129,11 @@ func DeleteHost(api libmachine.API) error {
 	m := util.MultiError{}
 	m.Collect(host.Driver.Remove())
 	m.Collect(api.Remove(constants.MachineName))
+
+	if len(m.Errors) == 0 && glog.V(2) {
```
Comment (ulyssessouza) [inline pkg/minikube/cluster/cluster.go]: Notice that by placing it here, the same applies to `minikube delete`. I just preferred this to make the VM deletion messages coherent, independently of the trigger.

Comment (LalatenduMohanty) [reply pkg/minikube/cluster/cluster.go]: This is how it looks now and it does not look good as there is a repetition to about VM getting deleted. Also the whole output needs to be fixed as `Deleted: Minishift VM 'minishift'` should be `Minishift VM 'Minishift' deleted` to match it with other messages in `minishift delete`.  
```
$ minishift delete --v 5
You are deleting the Minishift VM: 'minishift'. Do you want to continue [y/N]?: y
Deleting the Minishift VM...
Deleted: Minishift VM 'minishift'
Minishift VM deleted.
```
However I would suggest we just change the `profile delete` command output in this PR.

Comment (ulyssessouza) [reply pkg/minikube/cluster/cluster.go]: Right. I'm gonna move this back to `profile delete` only

File: pkg/minikube/cluster/cluster.go [hunk 2]
```diff
@@ -129,6 +129,7 @@ func DeleteHost(api libmachine.API) error {
 	m := util.MultiError{}
 	m.Collect(host.Driver.Remove())
 	m.Collect(api.Remove(constants.MachineName))
+
```
Comment (LalatenduMohanty) [inline pkg/minikube/cluster/cluster.go]: Please remove this extra white line.

Comment (budhrg) [reply pkg/minikube/cluster/cluster.go]: Agreed. Looks like unnecessarily got added.

**Comment by LalatenduMohanty** [comment]
Removed the blank line and merged https://github.com/minishift/minishift/commit/bcfd3a5dc1acb48c42555ad8d401c0d680d89b11

**Comment by LalatenduMohanty** [comment]
@ulyssessouza Thank you for the fix.

File: pkg/minikube/cluster/cluster.go (hunk 2 above)
Comment (ulyssessouza) [reply pkg/minikube/cluster/cluster.go]: Ops.... My bad...

**Pull request closed without merging**